dashboard/src/components/IasqlEditor.tsx

```diff
@@ -45,7 +45,7 @@ export default function IasqlEditor() {
 
   // Handlers
   const getInitialQuery = useCallback((sql: string | null) => {
-    let initialQuery = 'SELECT * FROM iasql_help();';
+    let initialQuery = editorTabs?.[editorSelectedTab]?.content || 'SELECT * FROM iasql_help();';
```

Comment:
Super minor nit. You're technically supposed to use `??` here instead of `||` since `false || 'something'` will become `'something'` while `false ?? 'something'` will be `false`, and only if the value is `null` or `undefined` will it fall through to the backup value.

Reply:
yes, that's better, lets's fix it!
```suggestion
    let initialQuery = editorTabs?.[editorSelectedTab]?.content ?? 'SELECT * FROM iasql_help();';
```